<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>motorWidget</class>
 <widget class="QWidget" name="motorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="positionControl">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>4</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="resetControl">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>0</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset limit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seekControl">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>0</y>
     <width>56</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Seek</string>
   </property>
  </widget>
  <widget class="QPushButton" name="centerControl">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>0</y>
     <width>86</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Set center</string>
   </property>
  </widget>
  <widget class="QPushButton" name="limitControl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>86</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Set limit</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nullControl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>86</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Set NULL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="powerControl">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>101</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Power: on</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
